routes/game.js

```diff
@@ -0,0 +1,9 @@
+let getGameData = (req, res) => {
+  console.log('User: ' + req.user.id);
```

Comment:
I know more is going to go here, but what is game data? How is it different than level data?

Reply:
Game data will be the data generated by the game.  The same data we discussed this morning for statistics.